app/components/provider_interface/status_box_components/awaiting_provider_decision_component.html.erb

```diff
@@ -0,0 +1,3 @@
+<%= render SummaryListComponent.new(rows: rows) %>
```

Comment:
Why doesn't this one have a title like all the others? It looked out of step with the others when I went through them on the review app - https://apply-for-te-1635-imple-g8dt6g.herokuapp.com/rails/components/provider_interface/status_box_component/awaiting_provider_decision

Reply:
This is intentional / the current design. I'm guessing this is the first status that appears within the provider UI, anything past that point corresponds to a provider or candidate action we would like to highlight.